CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Yii Framework 2 apidoc extension Change Log
 3.0.4 under development
 -----------------------
 
-- no changes in this release.
+- Bug #282: Fix `BaseDoc::extractFirstSentence()` (WinterSilence)
```

Comment:
Please describe what exactly is fixed.

Reply:
@arogachev we document changes in "dev. only" tools(`build/`)?